src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts

```diff
@@ -316,27 +276,41 @@ export const getNotificationResponses = async (
   translations: AnyRecord,
   req: AppRequest
 ): Promise<any> => {
+  const nonAdminRespondCollection = sendNotificationType?.respondCollection
+    ? sendNotificationType.respondCollection
+    : [];
```

Comment:
minor: can be simplified with just `const nonAdminRespondCollection = sendNotificationType?.respondCollection || []`

Reply:
Done